src/components/flame-graph/Canvas.js

```diff
@@ -168,6 +174,15 @@ class FlameGraphCanvasImpl extends React.PureComponent<Props> {
       },
     } = this.props;
 
+    const { hoveredItem } = hoverInfo;
+
+    const { cssToUserScale } = scale;
+    if (cssToUserScale !== 1) {
+      throw new Error(
+        'FlameGraphCanvasImpl sets scaleCtxToCssPixels={true}, so canvas user space units should be equal to CSS pixels.'
+      );
+    }
```

Comment:
Are these checks useful? After all, this component controls the prop `scaleCtxToCssPixels`, so is there a risk that in the future this might break?

(removing them is optional, I'm defering to your opinion, but I'm just curious about your thoughts)